packages/xforms-engine/src/lib/dom/query.ts

```diff
@@ -3,7 +3,7 @@ import type {
 	KnownAttributeLocalNamedElement,
 	LocalNamedElement,
 } from '@getodk/common/types/dom.ts';
-import type { SelectElement } from '../../parse/body/control/select/SelectDefinition';
+import type { SelectElement } from '../../parse/body/control/SelectControlDefinition.ts';
```

Comment:
Question:

Why do we have one for Select, while not for Range, Note, Inputs? 
Looking to understand the reason so I can see if I need it for Rank or Geopoint